### DISCUSSION TITLE
android usage question

### DISCUSSION THREAD

**Discussion by schoko11** [djipco/webmidi#165] [Q&A] [answered]
Hi, this project looks promising!

I made a somewhat prototype with open stage control to control my hardware with midi.
https://github.com/schoko11/KEKO-Kemper-Control

My goal would be to use a web browser to control the hardware either from windows or android (/iphone)...

Can i use  webmidi.js via bluetooth / wlan (with a bluetooth midi cable) ? 

Thanks, klaus

**Comment by djipco** [comment, marked as answer]
If the device is detected at the OS level, it should also be detected in the browser. Bluetooth should work fine. The Android browser supports the Web MIDI API.

**Comment by schoko11** [comment]
Thanks for the quick answer, i will opt for cme widi master. My midi hardware is on their compatibility list. So "just" programmng is to do -> LOL

Should i start with version 3? (it will take a while before i beginn)..

**Reply by djipco** [reply]
Version 3 is currently in alpha. It should have stabilized in a month or so. Hopefully, we will be in beta before November. If you can stomach a little transitory instability, you might as well start with version 3.
